lightly/transforms/rotation.py

```diff
@@ -50,10 +50,30 @@ def __call__(self, sample):
         return sample
 
 class RandomRotateDegrees(object):
+    """Random rotate image between two rotation angles with a random probability.
```

Comment:
There is no need to inherit from `object` when defining a new class:
```
class RandomRotateDegrees(object):
```
You can just write it as:
```
class RandomRotateDegrees:
```

Inheritance from object was only required in the old python days. We sadly still have some of this old code around, as you can see in the `RandomRotate` class. Feel free to remove it from there as well.